src/System.IO.FileSystem.AccessControl/dir.props

```diff
@@ -3,5 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
No need to copy it, just move it to common.  Let's move the discussion over to #15337